### PR TITLE
Update tests against Symfony 3.1

### PR DESCRIPTION
**Pull request by romainneutron**

### CODE REVIEW

**Comment by romainneutron** [comment]
Should be rebase once #1377 is merged

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,6 @@ matrix:
         - php: 5.6
           env: TWIG_VERSION=2.0
         - php: 5.6
-          env: SYMFONY_DEPS_VERSION=3.0
+          env: SYMFONY_DEPS_VERSION=3.1
```
Comment (fabpot) [inline .travis.yml]: we should keep the 3.0 version in the matrix in addition to the new 3.1.

**Comment by fabpot** [comment]
Can you rebase?

**Comment by romainneutron** [comment]
PR has been rebased

**Comment by SpacePossum** [comment]
Travis fails on 5.6, https://travis-ci.org/silexphp/Silex/jobs/133305263#L453
not sure if related?

**Comment by romainneutron** [comment]
PR updated

File: tests/Silex/Tests/Provider/MonologServiceProviderTest.php [hunk 2]
```diff
@@ -43,7 +44,12 @@ public function testRequestLogging()
         $this->assertTrue($app['monolog.handler']->hasDebug('< 200'));
 
         $records = $app['monolog.handler']->getRecords();
-        $this->assertContains('Matched route "GET_foo"', $records[0]['message']);
+        if (version_compare('3.1', Kernel::VERSION, '>=')) {
```
Comment (fabpot) [inline tests/Silex/Tests/Provider/MonologServiceProviderTest.php]: We are using `if (Kernel::VERSION_ID >= 30100) {` elsewhere.

**Comment by fabpot** [comment]
Thank you @romainneutron.

File: tests/Silex/Tests/Provider/MonologServiceProviderTest.php (hunk 2 above)
Comment (fabpot) [inline tests/Silex/Tests/Provider/MonologServiceProviderTest.php]: done

Comment (romainneutron) [inline tests/Silex/Tests/Provider/MonologServiceProviderTest.php]: we did it at the same time !

**Pull request merged**